docs/library/network.WLAN.rst

```diff
@@ -130,7 +130,7 @@ Methods
    hidden         Whether SSID is hidden (boolean)
    security       Security protocol supported (enumeration, see module constants)
    key            Access key (string)
-   dhcp_hostname  The DHCP hostname to use
+   hostname       The station(access point) hostname to use
```

Comment:
maybe make it clear that it sets both, eg "The hostname for the station and access point interfaces"